gamification/src/index.js

```diff
@@ -12,16 +12,18 @@ import {getCookie} from './Functions/Cookies.js'
 import Login from './Components/Login/Login';
 import Navbar from './Components/Navbar/Navbar';
 
+import Group from './Components/Group/Group';
+
 // Student-only components
 import Courses from './Components/Courses/Courses';
-import Group from './Components/Group/Group';
 import StudentProfile from './Components/Profile/Profile';
 
 // Instructor-only components
 import TeacherPeriods from './Components/TeacherPeriods/TeacherPeriods';
 import TeacherMissions from './Components/TeacherMissions/TeacherMissions';
 import GradeStudent from './Components/GradeStudent/GradeStudent';
 import AddStudent from './Components/AddStudent/AddStudent';
+import TeacherGroups from './Components/TeacherGroups/TeacherGroups';
```

Comment:
refactor name to subject isntead of teach groups

Reply:
Changed to ``Subjects``